src/openforms/authentication/contrib/yivi_oidc/migrations/0005_data_migrate_set_attributegroup_uuid.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.23 on 2025-08-21 08:42
+import uuid
+
+from django.db import migrations
+from django.db.migrations.state import StateApps
+from django.db.models import Q
+
+
+def transform_set_attributegroup_uuid(apps: StateApps, _):
+    AttributeGroup = apps.get_model("yivi_oidc", "Attributegroup")
+
+    # Get a set of uuid that already exist
+    occupied_uuids: set = set(
+        AttributeGroup.objects.filter(~Q(uuid=None)).values_list("uuid", flat=True)
+    )
+
+    attribute_groups = AttributeGroup.objects.filter(uuid=None)
+
+    for attribute_group in attribute_groups:
+        attribute_group_uuid = uuid.uuid4()
+
+        # Check if the uuid is unique, if not generate a new one and check again
+        while attribute_group_uuid in occupied_uuids:
+            attribute_group_uuid = uuid.uuid4()
+
+        # Add the new uuid to the `occupied` set and set the attribute group uuid
+        occupied_uuids.add(attribute_group_uuid)
+        attribute_group.uuid = attribute_group_uuid
+
+    # Update the attribute groups
+    AttributeGroup.objects.bulk_update(attribute_groups, fields=["uuid"])
+
+
+def reverse_set_attributegroup_uuid(apps: StateApps, _):
```

Comment:
no point implementing this, you can just use `migrations.RunPython.noop` instead.

When you reverse migrations, the reverse from the previous migration (adding a field) is to drop the UUID field again, so the data will be gone anyway.